### PR TITLE
Add labels to created pull requests #61

### LINKED ISSUE #61: Add labels to created PRs
It would be great if we could label the PRs created by these actions. It would be way easier to find all the PRs for a single environment

### PR DESCRIPTION
**Pull request by gezapeti**
This resolves #61

### CODE REVIEW

**Comment by gezapeti** [comment]
I don't thing anything in the failed actions is related to the change. It looks like some access issue.

File: src/commit-and-pull-request.sh [hunk 1]
```diff
@@ -118,6 +118,11 @@ if [[ "${PROMOTION_METHOD}" == "pull_request" ]]; then
     gh pr view
   fi
 
+  if [[ "${LABELS}" ]]; then
```
Comment (highb) [inline src/commit-and-pull-request.sh]:
```suggestion
  if [[ -n "${LABELS}" ]]; then
```

Comment (highb) [reply src/commit-and-pull-request.sh]: Should check for non-null string here with -n.

**Review by highb — CHANGES_REQUESTED** [review]

**Comment by highb** [comment]
I'll merge this and test it out before releasing, since the github token seems to be unavailable from a fork.

**Comment by highb** [comment]
Ended up making an additional change over in #62 , so I'll close this PR. I'll be releasing this change as v4.2.0. 🥳 

Thank you for the contribution @gezapeti !

**Pull request closed without merging**